test/test_proxy_tensor.py

```diff
@@ -1108,6 +1108,20 @@ def forward(self, y_1, x_1):
     index_select = torch.ops.aten.index_select.default(y_1, 1, repeat_interleave);  y_1 = repeat_interleave = None
     return index_select""")
 
+    def test_mod_gcd_unbacked(self):
+        def f(x):
+            a, b, stride = x.tolist()
+            torch._check_is_size(a)
+            torch._check_is_size(b)
+            torch._check_is_size(stride)
+            ta = torch.randn(a * stride)
+            tb = torch.randn(b * stride)
+            r = torch.cat([ta, tb])
+            return r.view(a + b, stride)
+
+        r = str(make_fx(f, tracing_mode="symbolic")(torch.tensor([30, 20, 10])).code).strip()
+        self.assertExpectedInline(r, "")
```

Comment:
You'll need to update this so that it has the actual new output. But o/w it looks like it should work.